src/locale/translations/fr.json

```diff
@@ -214,7 +53,34 @@
         }
       }
     },
+    "ChooseRegionCTA": "Sélectionnez une province ou un territoire",
+    "How": "Diagnosed with COVID-19?",
+    "CTA": "Prochaines étapes",
+    "HowUrl": "https://www.canada.ca/",
     "ExposedHelpCTA": "Que faire en cas d’exposition",
+    "DiagnosedView": {
+      "Title": "Merci d’aider à lutter contre la COVID",
+      "Body1": {
+        "One": "Tomorrow, you’ll get your last notification asking permission to upload any new random codes your phone sent out.",
```

Comment:
This needs to be translated to french

Reply:
Checked to see if this exists in the code and it does.

```
export function pluralizeKey(baseKey: string, number: number) {
  if (number === 1) return `${baseKey}.One`;
  return `${baseKey}.Other`;
}
```